judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/TestStaticQueriesWithDefaultValues.java

```diff
@@ -0,0 +1,205 @@
+     *
+     *  . Create and save a MyEntity instance with the following field values. (*e2*)
+     *  (for Snapshot1.fldMyEntity001 and setOfMyEntities)
+     *    * fldBool = true
+     *    * fldDate = `1900-01-01`
+     *    * fldTime = `12:00:13`
+     *    * fldTimestamp = `2020-01-01T01:11:13-12:00`
+     *    * fldLong = 1234567890
+     *    * fldString = "AAA"
+     *    * fldDecimal = -1.2302
+     *    * fldEnum = MyEnum#A03
+     *
+     *  . Create and save a MyEntity instance with the following field values. (*e3*)
+     *  (for Snapshot1.fldMyEntity002, Snapshot1.fldMyEntity003, Snapshot1.fldMyEntity004)
+     *    * fldBool = false
+     *    * fldDate = Timestamp!now()!date()
+     *    * fldTime = `15:15:01`
+     *    * fldTimestamp = `2020-01-01T01:11:13+01:00`
+     *    * fldLong = 12345678999
+     *    * fldString = "AAA"
+     *    * fldDecimal = -2.2302
+     *    * fldEnum = MyEnum#A02
+     *
+     *  . Create and save a MyEntity instance with the following field values. (*e4*)
+     *  (for Snapshot1.fldMyEntity003, fldMyEntity005)
+     *    * fldBool = true
+     *    * fldDate = `2023-01-01`
+     *    * fldTime = `15:20`
+     *    * fldTimestamp = `2020-01-01T01:11:13+01:00`
+     *    * fldLong = -16
+     *    * fldString = "Lorem ipsum"
+     *    * fldDecimal = 999999999.9999
+     *    * fldEnum = MyEnum#A01
+     *
+     *  . Create and save a Snapshot1 instance without any field values. (*s1*)
+     *
+     *  . Check the value of the fields of the created *s1* instance. All of the following boolean expressions must be true.
+     *    * s1.fldMyEntity001 == e2
+     *    * s1.fldMyEntity002 == e3
+     *    * s1.fldMyEntity003 == e3 or s1.fldMyEntity003 == e4
+     *    * s1.fldMyEntity004 == e3
+     *    * s1.fldMyEntity005 == e4
+     *    * s1.fldMyEntity006 == e2 or s1.fldMyEntity006 == e3 or s1.fldMyEntity006 == e4
+     *    * s1.defEntities   == [e2]
+     *    * s1.otherEntities == [e4]
+     *
+     *  . The test is passed if all steps have been completed with the specified results.
+     *
+     */
+    @Test
+    @TestCase("TC017")
+    @Requirement(reqs = {
+            "REQ-SYNT-001",
+            "REQ-SYNT-002",
+            "REQ-SYNT-003",
+            "REQ-SYNT-004",
+            "REQ-SYNT-005",
+            "REQ-TYPE-001",
+            "REQ-TYPE-002",
+            "REQ-TYPE-004",
+            "REQ-TYPE-005",
+            "REQ-TYPE-006",
+            "REQ-TYPE-007",
+            "REQ-TYPE-008",
+            "REQ-TYPE-009",
+            "REQ-MDL-001",
+            "REQ-MDL-003",
+            "REQ-ENT-001",
+            "REQ-ENT-002",
+            "REQ-ENT-004",
+            "REQ-ENT-005",
+            "REQ-ENT-009",
+            "REQ-ENT-011",
+            "REQ-EXPR-002",
+            "REQ-EXPR-004",
+            "REQ-EXPR-005",
+            "REQ-EXPR-006",
+            "REQ-EXPR-007",
+            "REQ-EXPR-008",
+            // TODO: JNG-4392 "REQ-EXPR-012",
```

Comment:
According to JIRA, this ticket is marked as done. Can you please check if requirement is available?